.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
```

Comment:
and you need to add the other package json files in this repo .... e.g. https://github.com/owncloud/web/blob/master/packages/web-app-files/package.json

Reply:
Oh, I thought that it will automatically scan for all packages also in nested folders... Will add them, thanks!

Reply:
Hmm, there is no array for setting multiple dirs. And creating separate ecosystems for each package seems a bit overkill. Looking at this issue https://github.com/dependabot/dependabot-core/issues/603#issuecomment-408228573 they mention this:

> If you use Lerna or Yarn workspaces then it will automatically pick up all of your packages and update them in a single PR (per dependency, rather than multiple PRs for the same dependency).

Workspaces are defined in the root package.json so I guess it should be able to pick them up.